src/main/java/no/unit/nva/cristin/institutions/GatewayResponse.java

```
@@ -76,10 +75,10 @@ private void generateDefaultHeaders() {
         Map<String, String> headers = new ConcurrentHashMap<>();
         headers.put(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON);
         final String corsAllowDomain = Config.getInstance().getCorsHeader();
```

I think it's better to read and verify config in the constructor of the handler. We want to fail fast, not wait for first request.